File: ClientApp/src/app/models/competition/update-league.model.ts
import { UpdateLeagueTeam } from './update-league-team.model';

export class UpdateLeagueModel {
    teamCount: number;
    isHomeAway: boolean;
    peerToPeerPlayCount: number;
    riseTeamCount: number;
    fallTeamCount: number;
    wonPoint: number;
    drawPoint: number;
    lostPoint: number;
    leagueTeams: UpdateLeagueTeam[];
}

File: Models/Competition/UpdateLeagueModel.cs
using System.Collections.Generic;

namespace champi.Models.Competition
{
    public class UpdateLeagueModel
    {
        public int TeamCount { get; set; }
        public bool IsHomeAway { get; set; }
        public int PeerToPeerPlayCount { get; set; }
        public int RiseTeamCount { get; set; }
        public int FallTeamCount { get; set; }
        public List<UpdateLeagueTeamModel> LeagueTeams { get; set; }
    }
}